daemon/network.go

```diff
@@ -298,6 +301,10 @@ func (daemon *Daemon) createNetwork(create types.NetworkCreateRequest, id string
 		return nil, err
 	}
 
+	daemon.pluginRefCount(driver, driverapi.NetworkPluginEndpointType, plugingetter.ACQUIRE)
```

Comment:
Outside the scope of this PR: all caps declarations are unusual in Go and clash with the typical style. I opened #29564 to change this. This PR can be merged first and then I'll rebase afterwards.